perf/istio/gracefulShutdown/templates/redeployer-cron.yaml

```diff
@@ -1,3 +1,31 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: graceful-shutdown-account
```

Comment:
Maybe rename this to "pod-controller" or something - pretty sure we'll have other tests that will use
this service account, no need to create one per test.

Reply:
Or 'redeployer' - to match the yaml name